modules/cas_cache/utils/utils_mpool.h

```diff
@@ -78,15 +74,17 @@ void *cas_mpool_new(struct cas_mpool *mpool, uint32_t count);
  *
  * @return Pointer to the new items
  */
-void *cas_mpool_new_f(struct cas_mpool *mpool, uint32_t count, int flags);
+void *env_mpool_new_f(struct env_mpool *mpool, uint32_t count, int flags);
 
 /**
  * @brief Free existing items of memory pool
  *
  * @param mpool CAS memory pool reference
  * @param items Items to be freed
  * @param count - Count of elements to be free
+ *
+ * @return Allocation was from mpool and was freed
```

Comment:
I think it returns true if allocation was freed regardless of mpool / vmalloc fallback.

Reply:
I removed the redundant info here to avoid confusion. Do you think it's better now?